cmd/ipam/main.go

```diff
@@ -102,14 +105,14 @@ func run(cmd *cobra.Command, _ []string) error {
 
 	// Get the rest config.
 	cfg := restcfg.SetRateLimiter(ctrl.GetConfigOrDie())
-	options.Config = cfg
```

Comment:
Is it ok here not to pass the config to options?

Reply:
Yes it wasn't used, I also removed it from the struct itself